tests/llvm/fork_env_test.py

```diff
@@ -90,9 +90,12 @@ def test_fork_before_reset(env: LlvmEnv):
     """Test that fork() before reset() starts an episode."""
     assert not env.in_episode
     fkd = env.fork()
-    assert env.in_episode
-    assert fkd.in_episode
-    assert env.benchmark == fkd.benchmark
+    try:
+        assert env.in_episode
+        assert fkd.in_episode
+        assert env.benchmark == fkd.benchmark
+    finally:
+        fkd.close()
```

Comment:
Just wondering, is the assertion fails does the error gets propagated past `finally` without catching exception?

Reply:
Yes, `finally` does not trap the error, it just ensures that `fkd.close()` is run even if there is an error. Demo:

```py
>>> try:
...     assert False
... finally:
...     print("Hi, finally!")
...
Hi, finally!
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
AssertionError
```